packages/relay/src/AuthContext.js

```diff
@@ -47,3 +47,24 @@ class Provider extends React.Component<Props, State> {
 }
 
 export default { Consumer, Provider };
+
+type PropsWithContext = {};
+
+export function withAuthContext(Component: React.ElementType) {
+  const WithAuthContext = (props: PropsWithContext) => (
+    <Consumer>
+      {({ accessToken, actions: { setAccessToken } }) => (
+        <Component
+          {...props}
+          accessToken={accessToken}
+          setAccessToken={setAccessToken}
+        />
+      )}
+    </Consumer>
+  );
+  // $FlowExpectedError: We need to pass on the navigationOptions if any, flow does not know about it, but a react component might have it
```

Comment:
Maybe here that's also because it is written as a function, and not as a class?

Reply:
No, it is the same in `WithStorage` HOC which is a class.